packages/sihl/sihl.0.1.10/opam

```diff
@@ -37,6 +37,7 @@ depends: [
   "alcotest-lwt" {>= "1.2.0" & < "3.0.0" & with-test}
   "cohttp-lwt-unix" {>= "2.5.1"& with-test}
   "odoc" {with-doc}
+  "mirage-crypto"
```

Comment:
I don't think this is the right fix here. `sihl` uses the `Mirage_crypto` module but only implicitly through `tls` and does not require it in its dune file.

The only fix possible here I think is to add `< "0.16.0"` to the `tls` dependency above